weechat-android/build.gradle

```diff
@@ -3,7 +3,7 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:2.2.3'
+        classpath 'com.android.tools.build:gradle:3.0.0-beta2'
```

Comment:
It looks like the build is failing due to the Gradle version being changed. Will the build still work with the gradle version and `gradle-wrapper.properties` reverted?